.github/workflows/CI.yml

```diff
@@ -27,7 +27,6 @@ jobs:
           - GalacticNOMAD
           - GalacticNonconvex
           - GalacticOptimJL
-          - GalacticQuadDIRECT
```

Comment:
No, we just need to direct dev this one.

Reply:
Yes, I was being dumb, fixed now